colossalai/zero/gemini/chunk/chunk.py

```diff
@@ -61,12 +61,14 @@ class Chunk:
     def __init__(
         self,
         chunk_size: int,
-        process_group: ProcessGroup,
+        zero_group: ProcessGroup,
         dtype: torch.dtype,
         init_device: Optional[torch.device] = None,
         cpu_shard_init: bool = False,
         keep_gathered: bool = False,
         pin_memory: bool = False,
+        ddp_group: ProcessGroup = None,
+        use_ddp: bool = False,
```

Comment:
Only one new argument is needed. If `ddp_group` is None, ddp won't be used.

Reply:
ddp_group is obtained from the group mesh and it never return None. use_ddp is required.